tensorflow_addons/image/tests/gaussian_filter_ops_test.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+""" Tests for gaussian blur. """
+
+
+from tensorflow_addons.image.gaussian_filter_ops import gaussian_blur
+import numpy as np
+import numpy.core.multiarray
+from scipy.ndimage import gaussian_filter
+import tensorflow as tf
+import sys
+import pytest
+
+
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
+def test():
```

Comment:
Test functions should start with `test_`, you can take a look at our new test guide: https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md#writing-tests

Reply:
Done